problems/20.Count Days Spent Together.gs

```diff
@@ -1,47 +1,47 @@
 function countDaysTogether(arriveAlice: string, leaveAlice: string, arriveBob: string, leaveBob: string): number {
 
-    let Ama = parseInt(arriveAlice.substring(0, 2))
-    let Ada = parseInt(arriveAlice.substring(3, 5))
-    let Aml = parseInt(leaveAlice.substring(0, 2))
-    let Adl = parseInt(leaveAlice.substring(3, 5))
-    let Aaday: number = danUgodini(Ama, Ada)
-    let Alday: number = danUgodini(Aml, Adl)
-    let Adays: number[] = []
+    let aliMonArr = parseInt(arriveAlice.substring(0, 2))
+    let allDayArr = parseInt(arriveAlice.substring(3, 5))
+    let aliMonLea = parseInt(leaveAlice.substring(0, 2))
+    let allDayLea = parseInt(leaveAlice.substring(3, 5))
+    let aliArr365: number = dayOfYear(aliMonArr, allDayArr)
+    let aliLea365: number = dayOfYear(aliMonLea, allDayLea)
+    let ali365days: number[] = daysStaying(aliArr365, aliLea365)
```

Comment:
daysStaying should include all the above calculation, from line 3 up until 8. As they are the same for alice and bob correct? So why have 6 variables here which are related to calculating daysStaying?